### PR TITLE
Increase displayed lines of title for related_articles, promotions and search result

### PR DESCRIPTION
**Pull request by myokoym**
Close bm-sms/daimon-news-misc#84

### CODE REVIEW

File: app/assets/stylesheets/themes/default/_article-summary.scss [hunk 1]
```diff
@@ -226,6 +226,15 @@ _::-webkit-full-page-media, _:future, :root .article-summary__data-item { // for
       }
     }
 
+    @include media($mobile) {
+      .article-summary {
+        .article-summary__title {
```
Comment (okkez) [inline app/assets/stylesheets/themes/default/_article-summary.scss]: How about `&__title`?

File: app/assets/stylesheets/themes/default/_article-summary.scss [hunk 2]
```diff
@@ -281,6 +290,15 @@ _::-webkit-full-page-media, _:future, :root .article-summary__data-item { // for
     }
   }
 
+  @include media($mobile) {
+    .article-summary {
+      .article-summary__title {
```
Comment (okkez) [inline app/assets/stylesheets/themes/default/_article-summary.scss]: ditto.

File: app/assets/stylesheets/themes/default/_article-summary.scss [hunk 3]
```diff
@@ -291,6 +309,15 @@ _::-webkit-full-page-media, _:future, :root .article-summary__data-item { // for
 
     .related_articles {
       border-top: 1px solid rgba($main-color, 0.2);
+
+      @include media($mobile) {
+        .article-summary {
+          .article-summary__title {
```
Comment (okkez) [inline app/assets/stylesheets/themes/default/_article-summary.scss]: ditto.

File: app/assets/stylesheets/themes/default/_article-summary.scss (hunk 1 above)
Comment (myokoym) [inline app/assets/stylesheets/themes/default/_article-summary.scss]: I think we can't use `&__title`. If we use it, the result is changed:

``` scss
  /* .article-summary .article-summary__title { ... } */
  .article-summary {
    .article-summary__title {
      ...
    }
  }

  /* .article-summary__title { ... } */
  .article-summary {
    &__title {
      ...
    }
  }
```

**Comment by mtsmfm** [comment]
:shipit:

**Comment by myokoym** [comment]
Thanks! :headphones:

**Pull request merged**